packages/react-diagrams-routing/.npmignore

```diff
@@ -1,4 +1,5 @@
 *
+!src/**/*
```

Comment:
Im just not sure that we actually want the src files. This sometimes messes with the typescript compiler as well.